docs/dev/cli/saucectl/init.md

```diff
@@ -196,118 +281,422 @@ You can specify a combination of device characteristics within this flag:
+- `fail`: Download artifacts for failed suites only. (default value)
+
+</div>
+
+### <span className="cli">--browser</span>
+
+<div className="cli-desc">
+<p><small>| REQUIRED |</small></p>
+
+The name of the browser in which to run tests.
+</div>
+
+### <span className="cli">--platform</span>
+
+<div className="cli-desc">
+<p><small>| REQUIRED |</small></p>
+
+A specific operating system and version on which to run the specified browser and test suite,
+such as Windows 11 or macOS 13.
+
+</div>
+
+### <span className="cli">--version</span>
+
+<div className="cli-desc">
+<p><small>| REQUIRED |</small></p>
+
+The version of TestCafe that is compatible with the tests defined in this configuration.
+
+</div>
+
+
+## XCUITest Options Summary
+
+<table id="table-cli">
+  <thead>
+    <tr>
+      <th width="30%">Key</th>
+      <th width="10%">Shorthand</th>
+      <th>Description</th>
+    </tr>
+  </thead>
+  <tbody>
+    <tr>
+     <td><span className="t-cli"><a href="#--app">--app</a></span></td>
+     <td></td>
+     <td>Path to the application under test.</td>
+    </tr>
+    <tr>
+     <td><span className="t-cli"><a href="#--artifacts-when">--artifacts-when</a></span></td>
+     <td></td>
+     <td>When to download artifacts.</td>
+    </tr>
+    <tr>
+     <td><span className="t-cli"><a href="#--device">--device</a></span></td>
+     <td></td>
+     <td>Real device to use for testing.</td>
+    </tr>
+    <tr>
+     <td><span className="t-cli"><a href="#--other-apps">--other-apps</a></span></td>
+     <td></td>
+     <td>Path to additional applications.</td>
+    </tr>
+    <tr>
+     <td><span className="t-cli"><a href="#--simulator">--simulator</a></span></td>
+     <td></td>
+     <td>The iOS simulator to use for testing.</td>
+    </tr>
+    <tr>
+     <td><span className="t-cli"><a href="#--test-app">--test-app</a></span></td>
+     <td></td>
+     <td>Path to the test application.</td>
+    </tr>
+  </tbody>
+</table>
 
-**Shorthand:** `-t `
 
+## XCUITest Option Details
+
+### <span className="cli">--app</span>
+
+<div className="cli-desc">
+<p><small>| REQUIRED |</small></p>
+
+The path to a valid mobile application to test.
 </div>
 
-### <span className="cli">--username</span>
+### <span className="cli">--artifacts-when</span>
 
 <div className="cli-desc">
-<p><small>| OPTIONAL | STRING |</small></p>
+<p><small>| OPTIONAL |</small></p>
 
-A valid Sauce Labs user account. If you have not set your authentication credentials as environment parameters or generated a `credentials.yml` file, this value is required.
+Specifies when and under what circumstances to download artifacts. Valid values are:
 
-**Shorthand:** `-u `
+- `always`: Always download artifacts.
+- `never`: Never download artifacts.
+- `pass`: Download artifacts for passing suites only.
+- `fail`: Download artifacts for failed suites only. (default value)
 
 </div>
 
-## Examples
+### <span className="cli">--device</span>
```

Comment:
Maybe worth while to link to the more in depth doc about device allocation (https://docs.saucelabs.com/mobile-apps/supported-devices/#static-and-dynamic-device-allocation)?

Reply:
Good idea! 👍